content/en/docs/setup/install/multicluster/single-network/primary/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Add a Primary Cluster
+description: Describes how to add a primary cluster to your mesh on a flat network.
+weight: 20
+keywords: [multicluster, primary-cluster, deployment]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+To build a {{< gloss >}}multicluster{{< /gloss >}} deployment, you can add
+{{< gloss "primary cluster" >}}primary clusters{{< /gloss >}} to your
+{{< gloss >}}service mesh{{< /gloss >}}.
+
+To give you clean instructions, this section starts right after you complete the
+[initial configuration](/docs/setup/install/multicluster/single-network/initial-configuration),
+and assumes that `Cluster_1` and `Cluster_2` reside on the same network.
+However, you can add primary clusters to your existing mesh regardless of your
+current [deployment model](/docs/ops/deployment/deployment-models/).
+
+Add primary clusters to your deployment to improve the availability of your
+mesh. To keep multiple {{< gloss "control plane" >}}control planes{{< /gloss >}}
+in sync, apply your configuration to each cluster. Production systems employ
+continuous integration and continuous deployment (CI/CD) pipelines to apply
+configurations since they typically have at least one control plane per region.
+Administrators use these pipelines to manage configuration rollouts. Apply new
+configurations to one primary cluster at a time to help minimize
+troubleshooting.
```

Comment:
This sounds like a Markov chain generated it. 😁

What I think you're trying to get across is:

- a primary cluster is one with a control plane replica deployed in it
- all multi-cluster meshes need at least one primary cluster
- you can choose to make subsequent clusters primary clusters or remote clusters
- multiple control planes can be in different regions
- you should apply config one at a time (pref with a CI/CD system)

I'm a little surprised there is no global config, and you might want to explain why not?

Reply:
When there's a global config, the paragraph can be revisited as a separate PR at a future date. Rewording the paragraph is possible but it was the result of multiple compromises already.